src/core/ext/filters/client_channel/lb_policy/grpclb/grpclb.c

```diff
@@ -324,6 +324,13 @@ typedef struct glb_lb_policy {
    * Otherwise, we delegate to the RR policy. */
   size_t serverlist_index;
 
+  bool fallback;
```

Comment:
It probably isn't necessary to have a bool for this.  Instead, we can just check whether `fallback_addresses` is non-NULL (i.e., we should only set this `fallback_addresses` field to a non-NULL value if we are given one or more backend addresses and we have not already gotten back a response from the balancer).